### ISSUE TITLE
[ML-5] Lightweight SE-based gain/pan predictor для soundcheck

### ISSUE THREAD

**Issue by tubeslave** [tubeslave/AUTO-MIXER-Tubeslave#5]
## Описание

Реализовать лёгкую нейросетевую модель на основе **Squeeze-and-Excitation (SE) блока** из Martínez-Ramírez et al. (ISMIR 2022) для предсказания gain и panning параметров при саундчеке.

### Архитектура (из статьи)

```
Input stems (2K, A) → Conv1D (N=128, size 64) → Conv1D-Local (N, size 128) 
→ MaxPool (64-point) → TCN (4 stacks × 6 layers, dilation 1..32) 
→ BLSTM (3 layers, hidden 64) → Unpool → Element-wise multiply with X1
→ SE block (Abs → Global Avg → FC(16N) → FC(N) → adaptive gain sc)
→ Conv1D.T → Summation → stereo output
```

SE-блок учится применять **adaptive gain** к каждому частотному каналу — аналог dynamic EQ + auto-panning.

### Применение в Automixer

- **Soundcheck mode**: подать 10 сек каждого инструмента → модель предсказывает начальные gain/pan/EQ параметры
- **Замена rule-based**: постепенно заменить статические профили на learned predictions
- **Cross-Adaptive**: использовать вместо IMP rule-based подхода в `cross_adaptive_eq.py`

## Задачи

- [ ] Реализовать архитектуру на PyTorch/ONNX (2.7M параметров — fits on M3 Max)
- [ ] Обучить на MUSDB18 (86 train + 14 val + 50 test songs) с preprocessing из Issue #4
- [ ] Экспортировать в ONNX/CoreML для inference на MacBook
- [ ] Реализовать inference pipeline: 10-сек audio frames @ 44.1kHz → predicted mixing mask
- [ ] Интегрировать в Soundcheck tab: кнопка "AI Suggest" → предлагает gain/pan настройки
- [ ] A/B тест: сравнить AI-suggested vs rule-based параметры на тестовых треках
- [ ] Loss function: stereo-invariant L_b (формула 8 из статьи) с A-weighting pre-emphasis

## Источник

- Martínez-Ramírez et al., ISMIR 2022, Sec. 2.3 (Architecture), Table 1, Sec. 2.4 (Loss)
- SE-block: Hu, Shen, Sun, "Squeeze-and-Excitation Networks", CVPR 2018
- Code: https://marco-martinez-sony.github.io/FxNorm-automix/

## Оценка трудоёмкости

2–3 недели

## Зависимости

- #1 (Per-instrument LUFS targets)
- #4 (EQ normalization pipeline)

## Связь с текущим кодом

Новый модуль `backend/ml_mixer.py`, интеграция с soundcheck workflow

**Comment by tubeslave** [comment]
Реализовано в рамках полной имплементации проекта (Phases 1-10). Commit 2f88d21 на main.
